services/apps/alcs/src/alcs/application-decision/application-decision-condition/application-decision-condition.dto.ts

```diff
@@ -107,6 +107,9 @@ export class ApplicationHomeDto {
   @AutoMap()
   applicant: string;
 
+  @AutoMap(() => ApplicationDecisionConditionTypeDto)
```

Comment:
How can an application type be mapped to an application decision condition type?